src/graph/IngestExecutor.cpp

```diff
@@ -0,0 +1,78 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "http/HttpClient.h"
+#include "graph/IngestExecutor.h"
+#include "process/ProcessUtils.h"
+#include "webservice/Common.h"
+#include "webservice/WebService.h"
+
+#include <folly/executors/Async.h>
+#include <folly/futures/Future.h>
+
+namespace nebula {
+namespace graph {
+
+IngestExecutor::IngestExecutor(Sentence *sentence,
+                               ExecutionContext *ectx) : Executor(ectx) {
+    sentence_ = static_cast<IngestSentence*>(sentence);
+}
+
+Status IngestExecutor::prepare() {
+    return checkIfGraphSpaceChosen();
+}
+
+void IngestExecutor::execute() {
+    auto *mc = ectx()->getMetaClient();
+    auto  addresses = mc->getAddresses();
+    auto  metaHost = network::NetworkUtils::intToIPv4(addresses[0].first);
+    auto  spaceId = ectx()->rctx()->session()->space();
+
+    auto func = [metaHost, spaceId]() {
+        std::string tmp = "http://%s:%d/%s?space=%d";
```

Comment:
static const char * is better?

Reply:
use `char *` could avoid to call `c_str()`